eva-lib/src/main/java/uk/ac/ebi/eva/lib/metadata/eva/RoCrateMetadataAdaptor.java

```diff
@@ -100,6 +102,26 @@ public RoCrateMetadata getMetadataByProjectAccession(String accession) {
         return new RoCrateMetadata(entities);
     }
 
+    public RoCrateMetadata getAllProjects(){
+        List<Project> projects = projectRepository.findAll();
+        // Construct the project-related entities
+        List<RoCrateEntity> projectsRoEntities = new ArrayList<>();
+        List<RoCrateEntity> allAdditionalProjectProperties = new ArrayList<>();
+
+        // Construct the DatasetEntity for the project and add all entities to the RO-crate metadata
+        for (Project project : projects) {
+            List<RoCrateEntity> additionalProjectProperties = getAdditionalProjectProperties(project);
+            allAdditionalProjectProperties.addAll(additionalProjectProperties);
+            projectsRoEntities.add(new DatasetMinimalProjectEntity(project.getProjectAccession(), project.getTitle(), project.getDescription(),
+                    getFirstSubmissionDate(project), getReferences(additionalProjectProperties)));
+        }
+        List<RoCrateEntity> entities = new ArrayList<>();
```

Comment:
```suggestion

        // Construct the DataCatalogEntity and add all entities to the RO-crate metadata
        List<RoCrateEntity> entities = new ArrayList<>();
```
If you want - it's a less complicated method than the single project one, so the comments grouping sections together are less important.